<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serch_student</class>
 <widget class="QDialog" name="Serch_student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serch</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sex</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>age</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>subject</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>serch</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
